src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -94,120 +88,35 @@ public NetworkStream(Socket socket, FileAccess access, bool ownsSocket)
         }
 
         // Socket - provides access to socket for stream closing
-        protected Socket Socket
-        {
-            get
-            {
-                return _streamSocket;
-            }
-        }
-
-        internal Socket InternalSocket
-        {
-            get
-            {
-                Socket chkSocket = _streamSocket;
-                if (_cleanedUp || chkSocket == null)
-                {
-                    throw new ObjectDisposedException(this.GetType().FullName);
-                }
-
-                return chkSocket;
-            }
-        }
-
-        internal void InternalAbortSocket()
-        {
-            if (!_ownsSocket)
-            {
-                throw new InvalidOperationException();
-            }
-
-            Socket chkSocket = _streamSocket;
-            if (_cleanedUp || chkSocket == null)
-            {
-                return;
-            }
-
-            try
-            {
-                chkSocket.Dispose();
-            }
-            catch (ObjectDisposedException)
-            {
-            }
-        }
-
-        internal void ConvertToNotSocketOwner()
-        {
-            _ownsSocket = false;
-            // Suppress for finialization still allow proceed the requests
-            GC.SuppressFinalize(this);
-        }
+        protected Socket Socket => _streamSocket;
 
         // Used by the class to indicate that the stream is m_Readable.
         protected bool Readable
```

Comment:
@stephentoub I don't think it matters too much but IMO having two things meaning exactly the same thing is redundant